README.md

```diff
@@ -310,6 +313,63 @@ OPTIONS:
    --output-format value  Either "table", "json" or "json-pretty" (default: "table")
 ```
 
+
+## job
+
+job specific commands
+
+```
+NAME:
+   nomad-helper job - job specific commands with a twist
+
+USAGE:
+   nomad-helper job stop [command options] [arguments...]
+
+COMMANDS:
+     stop   Stop will stop the job from with ability to purge from the nomad cluster
+     move   Move will add/append a constraint to the job that will resolve to moving the job
+
+OPTIONS:
+   --help, -h                                                 show help
+```
+
+### stop
+
+```
+USAGE:
+   nomad-helper job stop [job_name] [command options]
+
+OPTIONS:
+   --as-prefix  Filter jobs by their name with prefix matching api-
```

Comment:
would it make more since to be `with-prefix` as-prefix does not make too much sense, wondering if thats the case with other commands in the nomad helper, if it is then its fine

Reply:
copied the flag from `stop` command